tests/external.py

```diff
@@ -0,0 +1,12 @@
+# simulate an external function which cloudpickle would normally
+# only save a reference to
+mutable_variable = ["return_a_string"]
+mutable_variable2 = ["_nested"]
+
+
+def an_external_function():
+    return mutable_variable[0]
+
+
+def nested_function():
```

Comment:
I would rather name this `wrapping_function` or `outer_function` or `function_with_nested_call_to_another_function`.